desktop/components/commercial_filter/page_title.coffee

```diff
@@ -0,0 +1,24 @@
+_ = require 'underscore'
+{ capitalize } = require 'underscore.string'
+
+module.exports = (query, filters) ->
+  queriedFilter = {}
+  if query.medium
+    queriedFilter.medium = query.medium
+  else
+    queriedFilter.medium = '*'
```

Comment:
:+1:

Reply:
There's probably a ⛳️ /functional way of writing this helper.../shrug